test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -149,14 +151,26 @@ proc ::AlgorandGoal::StartNetwork { NETWORK_NAME NETWORK_TEMPLATE TEST_ALGO_DIR
         puts "network create $NETWORK_NAME"
         spawn goal network create --network $NETWORK_NAME --template $NETWORK_TEMPLATE --datadir $TEST_ALGO_DIR --rootdir $TEST_ROOT_DIR
         expect {
-            timeout { close; ::AlgorandGoal::Abort "Timed out creating network with timeout: $timeout" }
-            "^Network $NETWORK_NAME created under.*" { puts "Network $NETWORK_NAME created" ; close  }
-            close
+            timeout { close; ::AlgorandGoal::Abort "Timed out creating network" }
+            "^Network $NETWORK_NAME created under.*" { puts "Network $NETWORK_NAME created" ; exp_continue }
+            eof { ::AlgorandGoal::CheckEOF "Unable to create network" }
```

Comment:
What is the significance of this statement?

Reply:
This change is needed in order to verify the return code from the goal command.